extensions/msal-node-extensions/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Authentication Extensions for Node
+The Microsoft Authentication Extensions for Node offers secure mechanisms for client applications to perform cross-platform token cache serialization and persistence. It gives additional support to the Microsoft Authentication Library for Node (MSAL).
+
+MSAL Node supports an in-memory cache by default and provides the ICacheManager to perform cache serialization. Developers are required to implement their own cache persistance across multiple platforms and Microsoft Authentication Extensions makes this simpler.
+
```

Comment:
A little unclear. Shouldn't we say `msal-extensions-for-node` is a default implementation of cache persistence for developers who choose to not implement their own persistence?

Reply:
Updated